app/models/outgoing_email.rb

```diff
@@ -34,7 +34,8 @@ class OutgoingEmail < ApplicationRecord
   validates_presence_of :sent_at
   has_one_attached :attachment
 
-  after_create :record_outgoing_interaction, :deliver, :broadcast
+  after_create :deliver, :broadcast
+  after_create :record_outgoing_interaction, if: ->(email) { email.user.present? }
```

Comment:
I noticed it's a little bit complicated copy-pasta to remember to `if: ->(email) { email.user.present? }` here as well as similarly in text messages.

We discussed two different options while reviewing:

* Make an InteractionTrackingService so the complication is out of the model
* Make a SystemEmail model (but BG showed me that this doesn't solve all problems that a service does, so service is probably just better)